### ISSUE TITLE
Questions about the code

### ISSUE THREAD

**Issue by poo-pee-poo** [mesakarghm/LPRNET#5]
Hi, Thank you for creating this cool repository!

I had a couple questions about the code that I hope you can answer:

1. The original LPRNet paper specifies that it takes in a 94x24 image for the model input, which is a good size for license plates that are much wider than they are tall (like Chinese plates). In my case however, I'm using US plates which are generally taller and have a 2:1 aspect ratio.
Therefore, it probably would be more optimal for US plates to use an image size of 96x48. **I was wondering, does changing the model image size require any special changes to the neural network architecture?** (For example, changing the strides of some layers, etc)
3. In the original LPRNet paper, it says that before feeding into the main recognition network, it first goes through a Spatial Transformer Layer. I noticed in your blog post from 2021 it states that you instead will be using WPOD-NET to de-warp the image. **I was just wondering, how do the results compare between WPOD-NET and embedding an STN into the network? Was WPOD-NET more accurate for you?** 
From my experience, WPOD-NET was **much** slower, and I believe it can't process multiple plates in one image, which is why in the original [paper](https://openaccess.thecvf.com/content_ECCV_2018/papers/Sergio_Silva_License_Plate_Detection_ECCV_2018_paper.pdf), they crop out each car first.

I really appreciate any time you spend answering these questions, thank you so much!

**Comment by mesakarghm** [comment]
Hi,
Sorry for the late response, I'm not actively maintaining this repo anymore and I'm working on a similar implementation with PyTorch, with updated packages. 
Anyways, to answer your question, 
1: I believe changing the model image size doesn't require any special change to the neural network architecture. Earlier, I was able to train on bigger license plates without any change to the architecture. 
2: Yes, the paper says to use a Spatial Transformer Layer. I was not able to embed an STN into the network (for some reason the accuracy dropped extremely) which is why I went for WPOD-NET. Yes, WPOD-NET was much slower so yeah, if you can embed STN into the network without any drop in accuracy, that'd be a lot more useful. 
If you are successful to do so, you are welcome to send in a PR and I'd be happy to update the codebase for other users.
